libs/commands/publish/README.md

```diff
@@ -146,28 +145,6 @@ lerna publish from-package --git-head ${CODEBUILD_RESOLVED_SOURCE_VERSION}
 
 Under all other circumstances, this value is derived from a local `git` command.
 
-### `--graph-type <all|dependencies>`
```

Comment:
--graph-type is no longer supported. This was called out as deprecated in the code, giving a warning when used that it would be removed in Lerna v6.